scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -289,7 +289,7 @@ def setup_layout(self):
         runs_icon = icons.get_icon("mdi.play-circle-outline")
         _ = QListWidgetItem(runs_icon, "Runs", self.tab_choice_list)  # noqa
 
-        settings_icon = icons.get_icon("mdi.settings")
+        settings_icon = icons.get_icon("mdi.settings-outline")
```

Comment:
To stay in line with the other GUIs and general icon usage could we stick to "mdi.settings"
```suggestion
        settings_icon = icons.get_icon("mdi.settings")
```

Reply:
That's been changed back now